source/reference/verification.txt

```diff
@@ -0,0 +1,211 @@
+scripting :ref:`document comparisons <c2c-verify-doc-comp>`,
+and the :ref:`Migration Verifier <c2c-verify-migration-verifier>`.
+The specific method you use to verify your data depends on your application
+workload and the complexity of the data.
+
+Details
+=======
+
+.. _c2c-verify-doc-counts:
+
+Document Counts
+---------------
+
+The most basic method of verifying sync is by comparing the number of documents
+in each synced collection on the source cluster to the number on the
+destination cluster.
+
+Compare the results on each cluster:
+
+- If the document count is different,  not all changes synced to the
+  destination cluster.
+
+- If the document count is the same, the destination cluster has the same
+  number of documents. This doesn’t guarantee a successful sync of all changes
+  to those documents, except in cases where you have an insert-only workload.
+
+To verify that the source and destination cluster have the same number of
+documents, run the :method:`~db.collection.countDocuments` method for each synced
+collection on each cluster and ensure that they have the same number of
+documents.
+
+.. io-code-block::
+
+   .. input::
+      :language: javascript
+
+      db.us_accounts.countDocuments()
+
+   .. output::
+      :language: javascript
+
+      345
+
+
+.. _c2c-verify-hash-comp:
+
+Hash Comparison
+---------------
+
+You can verify sync by comparing MD5 hashes of collections synced from the
+source cluster to the destination cluster. While hash comparison does ensure
+that the destination cluster has received all changes from the source, the
+:dbcommand:`dbHash` command locks the cluster, preventing additional writes
+until its operations complete.
+
+.. note::
+
+   Hash comparison is not possible with sharded clusters. It also does not
+   work for standalone servers and replica sets that use MongoDB 4.4 or earlier
+   releases, since the field order can vary.
+
+To verify that the source and destination cluster have the same hashes, execute
+the :dbcommand:`dbHash` command for the synced collections on each cluster and
+compare the hash value in the ``md5`` field of the return documents.
+
+.. io-code-block::
+
+   .. input::
+      :language: javascript
+
+      db.runCommand({
+         dbhash: 1,
+         collections: [
+            "accounts.us_accounts",
+            "accounts.eu_accounts",
+            ...
+         ]}).md5
+
+   .. output::
+      :language: text
+
+      d41d8cd98f00b204e9800998ecf8427e
+
+.. _c2c-verify-doc-comp:
+
+Document Comparison
+-------------------
+
+You can verify sync by comparing documents on the source and destination
+clusters. Write a script that queries collections on the source cluster and
+then checks that the document exists with the same values on the destination
+cluster.
+
+.. _c2c-verify-migration-verifier:
+
+Migration Verifier
+------------------
+
+Migration Verifier connects to the source and destination clusters and performs
```

Comment:
[question]

I'm assuming that the verifier comes packaged with the Relational Migrator install, and doesn't require anything additional to run. If that's the case, maybe that point is worth clarifying?

Reply:
@jeff-allen-mongo, I can't speak to Relational Migrator since I'm not familiar with it.  The link the C2C team provided goes to a separate GitHub project, which is compiled through Go tooling.  I link to it in the "For more information" bit at the bottom, but can  add some installation instructions if you think it's needed.

Reply:
I don't think we need full installation instructions, but it might help to move the GitHub link higher up on the page and say that users should go there to learn how to install the migrator. What do you think?

Reply:
Added a link to the admonition calling out installation information.